tests/unit/utils/sparqlhandler/test_code_list_state.py

```diff
@@ -0,0 +1,31 @@
+from csvcubed.utils.sparql_handler.code_list_inspector import CodeListInspector
```

Comment:
Im pretty sure we dont need this test file anymore as we populated `test_code_list_inspector.py` instead